helm/hpcc/templates/dali.yaml

```diff
@@ -52,21 +57,24 @@ spec:
 {{- $_ := set $tmpDaliScope "sashaAccess" list -}}
 {{- range $sashaName, $_sasha := $dali.services -}}
 {{- $sasha := ($_sasha | default dict) -}}
+{{- $ssecretsCategories := list "system" -}}
 {{- if not $sasha.disabled -}}
 {{- $_ := set $sasha "name" $sashaName -}}
 {{- $_ := set $tmpDaliScope "sashaAccess" (concat $tmpDaliScope.sashaAccess (splitList " " (include "hpcc.getSashaServiceAccess" $sasha))) -}}
-{{- include "hpcc.addSashaVolumes" (dict "root" $ "me" $sasha) | indent 6 }}
+{{- if has "data" $tmpDaliScope.sashaAccess }}
+{{- $ssecretsCategories := append $ssecretsCategories "storage" -}}
+{{- end }}
+{{- include "hpcc.addSashaVolumes" (dict "root" $ "me" $sasha "secretsCategories" $ssecretsCategories) | indent 6 }}
+{{- include "hpcc.addSecretVolumes" (dict "root" $ "categories" $ssecretsCategories ) | indent 6 }}
```

Comment:
These have also been moved into the range, which is going to cause problems if >1 service defined.
They should be called once the sashaAccess list has been formulated, after the range as before.